Improvement/tx status notification

estebanmino opened the pull request:
<!--
Thanks for your contribution!

Please ensure that any applicable requirements below are satisfied before submitting this pull request. This will help ensure a quick and efficient review cycle.
-->

**Description**

This PR updates the way we handle notifications
https://recordit.co/jG8hBOsEk1
and transaction details
https://recordit.co/PxKqRVBPcO

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #1414

estebanmino committed:
simple not update

estebanmino committed:
text update

estebanmino committed:
wip

estebanmino committed:
delete old details

estebanmino committed:
delete old confirm

estebanmino committed:
almost done withtx details

estebanmino committed:
modal working

estebanmino committed:
modal title

estebanmino committed:
rm transfer element

estebanmino committed:
clean

estebanmino committed:
fix transfer

estebanmino committed:
transfer and payment channel

estebanmino committed:
decodeTransferFromTx

estebanmino committed:
decodeDeploymentTx

estebanmino committed:
decodeConfirmTx

estebanmino committed:
onpress

estebanmino committed:
status

estebanmino committed:
close on view web

estebanmino committed:
more cleanup

estebanmino committed:
payment

estebanmino committed:
showing not

estebanmino committed:
closer?

estebanmino committed:
comment

estebanmino committed:
tx details and not

estebanmino committed:
animated

estebanmino committed:
tx not

estebanmino committed:
enable access view on not

estebanmino committed:
animated

estebanmino committed:
rename

estebanmino committed:
using txnnot manager

estebanmino committed:
working

estebanmino committed:
receive

estebanmino committed:
rm unused

estebanmino committed:
rm logs

estebanmino committed:
handle browser not

estebanmino committed:
parse date

estebanmino committed:
handle asset details

estebanmino committed:
tx summary rename props

estebanmino committed:
Refactor names in details

estebanmino committed:
handle primary currency

estebanmino committed:
missing props

estebanmino committed:
Merge branch 'develop' into improvement/tx-status-notification

estebanmino committed:
almost there

estebanmino committed:
working but browser

estebanmino committed:
finally

estebanmino committed:
one more thing

estebanmino committed:
done

estebanmino committed:
snaps

estebanmino committed:
missing locales

estebanmino committed:
update ethereum address

estebanmino committed:
snaps

estebanmino committed:
Merge branch 'develop' into improvement/tx-status-notification

estebanmino committed:
handle instapay txs

estebanmino committed:
snaps

andrepimenta commented on app/core/TransactionsNotificationManager.js:
```diff
@@ -58,8 +59,8 @@ class TransactionsNotificationManager {
 
-			if (data && data.message && data.message.transaction && data.message.transaction.nonce) {
-				nonce = data.message.transaction.nonce;
+			if (data && data && data.transaction && data.transaction.nonce) {
```
`data` appears twice

estebanmino replied:
😅

andrepimenta reviewed (approved):
Just found little typo, after that I think it can be merged.
All the changes make sense to me!  👌

estebanmino committed:
feeless tx

estebanmino committed:
data check

estebanmino committed:
merge dev

estebanmino committed:
No fee

estebanmino committed:
instance._hideTransactionNotification

estebanmino committed:
fix instapay notifications

estebanmino committed:
elevation

estebanmino committed:
fix remaining issues

estebanmino committed:
apeed up cancel

estebanmino committed:
transaction modal

estebanmino committed:
speed cancel

estebanmino committed:
speedup cancel ui

estebanmino committed:
working

estebanmino committed:
added engine methods

estebanmino committed:
done

estebanmino committed:
snaps

estebanmino commented:
fixed issues and added this new behavior https://recordit.co/PFVoNwELc0 waiting for confirmation from @cjeria to start the review, I added some new components that need to be reviewed again

estebanmino committed:
Merge branch 'develop' into improvement/tx-status-notification

estebanmino committed:
develop|

estebanmino committed:
fix qaing

estebanmino committed:
fix ios build

estebanmino committed:
one snap

estebanmino commented:
@ibrahimtaveras00 this is ready to qa again

estebanmino committed:
remove test

estebanmino committed:
status text fix

estebanmino committed:
cancelled

estebanmino committed:
margin

estebanmino committed:
snaps

estebanmino committed:
fix insufficient funds

estebanmino committed:
doc

estebanmino committed:
Merge branch 'develop' into improvement/tx-status-notification

ibrahimtaveras00 reviewed (approved):
All fixes look good on both OS's, QA passed 👍
